Tests/UnitTests/Core/TrackFinder/CombinatorialKalmanFilterTests.cpp

```diff
@@ -125,10 +125,10 @@ struct MeasurementCreator {
         auto lResolution = vResolution->second.find(layerID);
         if (lResolution != vResolution->second.end()) {
           // Apply global to local
-          Acts::Vector2D lPos;
-          surface->globalToLocal(state.geoContext,
-                                 stepper.position(state.stepping),
-                                 stepper.direction(state.stepping), lPos);
+          auto lpResult = surface->globalToLocal(
+              state.geoContext, stepper.position(state.stepping),
+              stepper.direction(state.stepping));
+          Acts::Vector2D lPos = lpResult.value();
```

Comment:
```suggestion
          Acts::Vector2D lPos = surface->globalToLocal(
              state.geoContext, stepper.position(state.stepping),
              stepper.direction(state.stepping)).value();
```
I do think there the separate result is needed in this test.

Reply:
You have more or less the same in the KF test. Maybe we could add an ERROR printout here, just in case it ever fails?